react/Textarea/Textarea.js

```diff
@@ -1,9 +1,7 @@
 import styles from './Textarea.less';
-
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
-
```

Comment:
Interesting that your getting rid of the newlines between import types. I thought this was to make a distinction between them (global/local/css-module). I find it makes the file easier to read. Would love a lint rule for it personally.